impeller/entity/inline_pass_context.cc

```diff
@@ -165,7 +164,7 @@ InlinePassContext::RenderPassResult InlinePassContext::GetRenderPass(
   // Commands are fairly large (500B) objects, so re-allocation of the command
   // buffer while encoding can add a surprising amount of overhead. We make a
   // conservative npot estimate to avoid this case.
-  pass_->ReserveCommands(Allocation::NextPowerOfTwoSize(entity_count_));
+  // pass_->ReserveCommands(Allocation::NextPowerOfTwoSize(entity_count_));
```

Comment:
I should probably add this back for GLES and Metal for now and just make it virtual/no-op